tests/test_notifier.py

```diff
@@ -86,7 +80,17 @@ def room_api():
 
 @pytest.fixture
 def notifier():
-    return Notifier(server_address=SERVER_ADDRESS, server_api_token=SERVER_API_TOKEN)
+    notifier = Notifier(server_address=SERVER_ADDRESS, server_api_token=SERVER_API_TOKEN)
+
+    @notifier.on_server_notification
+    def handle_notification(_server_notification):
+        pass
+
+    @notifier.on_metrics
+    def handle_metrics(_metrics_report):
+        pass
```

Comment:
Why did we need to add this?

Reply:
I would double-check if notification tests does test enough cases to fail in case of API changes

Reply:
We didn't receive notifications since we didn't subscribe to them (it's what we're doing here)